src/AccessToken.php

```diff
@@ -266,20 +340,24 @@ private function retrieveCertsFromLocation($url, array $options = [])
         ), $response->getStatusCode());
     }
 
-    /**
-     * Set required defaults for JWT.
-     */
-    private function configureJwtService()
+    private function checkAndInitializePhpsec()
     {
-        $class = class_exists('Firebase\JWT\JWT')
-            ? 'Firebase\JWT\JWT'
-            : '\JWT';
+        // @codeCoverageIgnoreStart
+        if (!class_exists(RSA::class)) {
+            throw new \RuntimeException('Please require phpseclib/phpseclib v2 to use this utility.');
+        }
+        // @codeCoverageIgnoreEnd
+
+        $this->setPhpsecConstants();
+    }
 
-        if (property_exists($class, 'leeway') && $class::$leeway < 1) {
-            // Ensures JWT leeway is at least 1
-            // @see https://github.com/google/google-api-php-client/issues/827
-            $class::$leeway = 1;
+    private function checkSimpleJwt()
+    {
+        // @codeCoverageIgnoreStart
+        if (!class_exists(SimpleJWT::class)) {
+            throw new \RuntimeException('Please require kelvinmo/simplejwtlib ^0.2 to use this utility.');
```

Comment:
Why not require this? (If it stays in require-dev only, we need to add it to suggests)

Reply:
Because this would require the `openssl` PHP extension for all of our client libraries